### DISCUSSION TITLE
Why does html boolean attribute 'checked' become 'checked=""'?

### DISCUSSION THREAD

**Discussion by jraedisch** [parcel-bundler/parcel#6383] [Q&A]
Somewhere in the Pipeline, '=""' gets added to all html attributes.

I would like to disable this, since it is breaking my templates, because also '{{' becomes '{{=""' and so on.
Any ideas would be greatly appreciated.

I am using Parcel 2 and already deactivated all htmlnano modules.

**Comment by mischnic** [comment]
This is caused by posthtml-parser/render:

https://github.com/parcel-bundler/parcel/blob/62af2900cb23c2332f7ec069d81a47bdd78d621f/packages/transformers/html/src/HTMLTransformer.js#L20

https://github.com/parcel-bundler/parcel/blob/62af2900cb23c2332f7ec069d81a47bdd78d621f/packages/transformers/html/src/HTMLTransformer.js#L69

Example:

```js
const { default: parse } = require("posthtml-parser");
const { default: render } = require("posthtml-render");

const CODE = `<input type="checkbox" checked>`;

const ast = parse(CODE, {
	lowerCaseAttributeNames: true,
});

const output = render(ast);

console.log(output);
// <input type="checkbox" checked="">
```

As far as I can tell, there is no way to make them retain `checked` without value.

**Reply by jraedisch** [reply]
Thank you! I have just found https://github.com/posthtml/posthtml-boolean-attributes and will try it out. Is it not a common workflow to have parcel prepare templates (in my case for Go)? How else does one do that?
